content/03.results.md

```diff
@@ -227,21 +227,21 @@ Interestingly, none of the hypermutant samples showed evidence of signature 3 (p
-| PT_JNEV57VK               | BS_85Q5P8GF               | 7316-2594 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Radiation                          | Lynch syndrome                      | MSH2                        | 4.7  | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_HM5GFJN8               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome                      | MSH2                        | 35.9  | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_QWM9BPDY               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome                      | MSH2                        | 7.4  | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_P0QJ1QAH               | 7316-3058 | Progressive       | Solid Tissue      | Nivolumab                                        | Lynch Syndrome                      | MSH2                        | 6.3  | DMG, H3 K28, TP53 activated        |
-| PT_S0Q27J13               | BS_P3PF53V8               | 7316-2307 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Radiation              | None documented                     | None detected               | 15.5 | HGG, H3 wildtype, TP53 activated   |
-| PT_VTM2STE3               | BS_ERFMPQN3               | 7316-2189 | Progressive       | Derived Cell Line | Unknown                                          | Lynch Syndrome                      | None detected               | 5.7  | HGG, H3 wildtype, TP53 loss        |
-| PT_VTM2STE3               | BS_02YBZSBY               | 7316-2189 | Progressive       | Solid Tissue      | Unknown                                          | Lynch Syndrome                      | None detected               | 274.5 | HGG, H3 wildtype, TP53 activated   |
+| Kids First Participant ID | Kids First Biospecimen ID | CBTN ID   | Phase of therapy  | Composition       | Therapy post-biopsy                              | Cancer predisposition(s)       | Pathogenic germline variant             | TMB                | OpenPBTA molecular subtype         |
+|---------------------------|---------------------------|-----------|-------------------|-------------------|--------------------------------------------------|--------------------------------|-----------------------------------------|--------------------|------------------------------------|
+| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Temozolomide, CCNU, Radiation                    | None documented                | PMS2 p.S46I (LP)                        | 187.38765454966892 | HGG, H3 wildtype, TP53 activated   |
```

Comment:
Please fix significant figures for TMB.

Reply:
Ah, will do- I should have done this in the actual table.

Reply:
I think to avoid all of the manual changes every time this table gets updated, I'll  open a new PR for this in the analysis repo this afternoon and tag you for review so that we aren't doing double work there and here. Would be good to just paste the table every time.